src/components/SingleArticle.jsx

```diff
@@ -47,12 +79,26 @@ const SingleArticle = () => {
         <img src={singleArticle.article_img_url}></img>
         <button onClick={() => {
             voteUp();
+
         }}>voteUp</button>
         <button onClick={() => {
             voteDown();
+
         }}>voteDown</button>
+        <form onSubmit={handleSubmit}>
+            <label htmlFor="body">Post a new comment: </label>
+            <textarea
+            value={addComment || ""}
+            id="body"
+            onChange={handleChange}
+            />
+            <button>Post</button>
```

Comment:
At the moment, if we click 'post' twice, it'll post the message twice. Have a look at how we can disable the button if we're currently posting. 👍